site/src/pages/components/navigation-bar.explainer.mdx

```diff
@@ -0,0 +1,118 @@
+
+This allows authors to bring so-called "mega menus" to users with predictable behavior and good
+accessibility. A mega menu is a large, expandable navigation menu that displays multiple options in
+a panel when a user clicks or hovers over a navigation item. Unlike traditional dropdowns with a
+single column of links, mega menus organize content into multiple columns with subcategories, often
+incorporating visuals like images, icons, and text to provide an overview of a website's structure.
+
+**Note** that this explainer only covers navigation menus, not application menus, which are covered
+by the [Menu Elements Proposal](https://open-ui.org/components/menu.explainer/).
+
+## The `<navigationbar>` element
+
+The `<navigationbar>` element represents an in-page group of either links, or disclosure buttons
+that display a popover consisting of the navigation bar's links. It defaults to a horizontal
+orientation, and its child elements are flex items displayed in a row, however like the application
+menus proposal, we are discussing an `orientation` attribute to support horizontal navigation bars.
+See [Issue #1201](https://github.com/openui/open-ui/issues/1201) for more.
+
+This element:
+
+* Provides the implicit [`navigation`](https://w3c.github.io/aria/#navigation) ARIA role.
+* Provides the correct keyboard and focus behavior, drawing from both the
+  [Disclosure Navigation ~Menu~ Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/),
+  [Disclosure Navigation Hybrid
+  Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation-hybrid/),
+  and the the [Menubar Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/).
+
+## Accessibility
+
+One goal of this proposal is to make the menu elements we propose accessible by default. To achieve
+this in part, our proposal follows the ARIA Authoring Practices Guide (APG) for both the
+[Disclosure Navigation ~Menu~ Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/),
+[Disclosure Navigation Hybrid Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation-hybrid/),
+and best practices curated from accessibility experts as closely as possible; deviations are
+discussed with accessibility experts and documented here to the best of our ability. This section
+describes the concrete implications on our proposal.
+
+We acknowledge the longstanding accessibility issues with navigation menus, and our proposal is an
+attempt to bring accessible, feature-rich navigation bars to the web platform while avoiding the
+pitfalls that accessibility experts have raised when application menus are misused as navigation
+menus. See the Menu Elements proposal's
+[§ How does this proposal relate to navigation menus?](#how-does-this-proposal-relate-to-navigation-menus)
+section and [Issue #1193](https://github.com/openui/open-ui/issues/1193) for more information.
```

Comment:
I wonder if it would help to link to some public blog posts from a11y experts about these patterns here also?